pymongo/mongo_client.py

```diff
@@ -1127,10 +1126,9 @@ def close(self):
         sending one or more endSessions commands.
 
         Close all sockets in the connection pools and stop the monitor threads.
-        If this instance is used again it will be automatically re-opened and
-        the threads restarted unless auto encryption is enabled. A client
-        enabled with auto encryption cannot be used again after being closed;
-        any attempt will raise :exc:`~.errors.InvalidOperation`.
+        If this instance is used again it will NOT automatically be
+        re-opened and the threads restarted. No client can be used again after
+        being closed; any attempt will raise :exc:`~.errors.InvalidOperation`.
```

Comment:
I think we can remove the  "NOT automatically be re-opened and the threads restarted". Instead let's say:
"Once closed, the client cannot be used again and any attempt will raise  :exc:`~pymongo.errors.InvalidOperation`."

Please also add a versionchanged to this method.

Reply:
Done.